Source/WebCore/dom/Document.cpp

```diff
@@ -2198,7 +2189,7 @@ void Document::resolveStyle(ResolveStyleType type)
                 localMainFrame->eventHandler().dispatchFakeMouseMoveEventSoon();
 
         ++m_styleRecalcCount;
-        // FIXME: Assert ASSERT(!needsStyleRecalc()) here. Do we still have some cases where it's not true?
+        // FIXME: Assert ASSERT(!needsStyleRecalc()) here. fast/events/media-element-focus-tab.html hits this assertion.
```

Comment:
That might be because a version of media controls most platforms don't even use.